### PR TITLE
Correct resolver

### PR DESCRIPTION
**Pull request by prassee**
fixed cascading 2.6.1 resolve error by adding resolver
refer these urls
https://mvnrepository.com/artifact/com.twitter/scalding-core_2.11/0.17.4
https://mvnrepository.com/artifact/cascading/cascading-hadoop/2.6.1

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,6 +4,8 @@ version := "0.1"
 
 scalaVersion := "2.11.8"
 
-libraryDependencies += "com.twitter" %% "scalding-core" % "0.17.4"
+resolvers ++= Seq("abcd" at "http://conjars.org/repo/")
```
Comment (anant-indix) [inline build.sbt]: please change to "Conjars" instead of "abcd"

File: build.sbt [hunk 2]
```diff
@@ -4,6 +4,8 @@ version := "0.1"
...
 
-libraryDependencies += "org.apache.hadoop" % "hadoop-client" % "2.9.0" % Provided
+libraryDependencies ++= Seq(
+			"com.twitter" %% "scalding-core" % "0.17.4",
+			"org.apache.hadoop" % "hadoop-client" % "2.9.0" % Provided)
```
Comment (anant-indix) [inline build.sbt]: give a new line after Provided

**Review by anant-indix — COMMENTED** [review]
Rest LGTM

File: build.sbt (hunk 1 above)
Comment (prassee) [reply build.sbt]: addressed in new commit

**Pull request merged**